README.md

```diff
@@ -25,6 +25,23 @@ My common Iptables configuration:
 
 ![Iptables](assets/iptables.png)
 
+## Netstat
+Print network connections
+
+List tcp connections (t), without resolving ip addresses (n) and showing the process pid (p)
+```
+# netstat -tnp
```

Comment:
I was talking about this character `#`, indicating the bash command line. I'm using the `$` instead for this purpose. :smile: